### PR TITLE
[Converter/CodeClean] unnecessary memory map

### PR DESCRIPTION
**Pull request by jaeyun-jung**
1. remove unnecessary memory map in chain function.
2. failure case when configuring multi tensors.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

**Review by wooksong — APPROVED** [review]
Neat!

**Review by dongju-chae — APPROVED** [review]
LGTM

File: gst/nnstreamer/tensor_converter/tensor_converter.c [hunk 1]
```diff
@@ -838,46 +838,50 @@ _gst_tensor_converter_chain_timestamp (GstTensorConverter * self,
...
+    idx = 0;
+    buffer = gst_buffer_new ();
+    mem = gst_buffer_get_all_memory (buf);
 
-      idx = 0;
-      buffer = gst_buffer_new ();
+    for (i = 0; i < self->tensors_info.num_tensors; ++i) {
+      size = gst_tensors_info_get_size (&self->tensors_info, i);
+      gst_buffer_append_memory (buffer, gst_memory_share (mem, idx, size));
+      idx += size;
```
Comment (dongju-chae) [inline gst/nnstreamer/tensor_converter/tensor_converter.c:863]: It's not related to this PR but I think `idx` is confusing. Better to use `offset`?

Comment (jaeyun-jung) [reply gst/nnstreamer/tensor_converter/tensor_converter.c]: updated

**Pull request merged**